help/destinations/catalog/advertising/amazon-ads.md

```diff
@@ -123,6 +123,7 @@ The [!DNL Amazon Ads] connection supports hashed email address and hashed phone
 * To map hashed email addresses, select the `Email_LC_SHA256` identity namespace as a source field.
 * To map hashed phone numbers, select the `Phone_SHA256` identity namespace as a source field.
 * To map unhashed email addresses or phone numbers, select the corresponding identity namespaces as source fields, and check the `Apply Transformation` option to have Platform hash the identities on activation.
+* We require mapping of to a field containing countryCode value in the 2-character ISO format in order to facilitate the Identity resolution process.  (e.g., US, GB, MX, CA, etc.) Connections without countryCode mappings will result in negative impact to identity match rates
```

Comment:
slight rephrasing suggestion:

Amazon Ads requires you to map a field containing a `countryCode` value in the 2-character ISO format in order to facilitate the Identity resolution process (for example: US, GB, MX, CA, and so on). Connections without `countryCode` mappings will result in negative impact to identity match rates.

Reply:
@epaulser - reminder about this small item to address. Thank you!